decentraland/unity-explorer
# Chore: explore panel tech debts

## What does this PR change?

This PR adds animations for the explore panel.
It also fixes #769 and #812 

...

## How to test the changes?

1. Launch the explorer
2. Play with the explore panel
3. Observe the nice animations
4. Verify that navmap is all interactable even after teleporting around

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
